tiny_ai_client/models.py

```diff
@@ -87,7 +87,7 @@ def get_llm_client_wrapper(
 
     @property
     def model_name(self) -> str:
-        return self._model_name
+        return self.model_name
```

Comment:
tks for finding actually my mistake. the attribute should be `_model_name`. Can we fix that before merging.